mautic/mautic
# Add a way to directly modify the html code in Advanced mode (fix an issue)

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [yes]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [x]
| BC breaks? (use the c.x branch)        | [x]
| Automated tests included?              | [x] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR fixes the issues with the builder button. When you switch to the code mode and then cancel in the modal the builder button remains disabled

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new email (landing page) or choose the existing one
3. Change to code mode
    ![image](https://user-images.githubusercontent.com/96085911/167128381-0bc7c608-0cdf-4778-8f79-9611fec42e56.png)
4. Click "Cancel"
5. The builder button is enabled (the builder button was disabled before this PR)




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11153"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
